fetch_api/src/fetch_api/arm.py

```diff
@@ -76,6 +76,19 @@ def __init__(self):
         self._move_group_client.wait_for_server(rospy.Duration(10))
         self._compute_ik = rospy.ServiceProxy('compute_ik', GetPositionIK)
         self._tf_listener = TransformListener()
+        self.move_group = MoveGroupInterface(ARM_GROUP_NAME, "base_link")
+
+    """
+    Tucks arm
+    """
+    def tuck(self):
+        joints = ["shoulder_pan_joint", "shoulder_lift_joint", "upperarm_roll_joint",
+                  "elbow_flex_joint", "forearm_roll_joint", "wrist_flex_joint", "wrist_roll_joint"]
+        pose = [1.32, 1.40, -0.2, 1.72, 0.0, 1.66, 0.0]
+        while not rospy.is_shutdown():
+            result = self.move_group.moveToJointPosition(joints, pose, 0.02)
+            if result.error_code.val == MoveItErrorCodes.SUCCESS:
```

Comment:
If it fails once how likely is it to succeed with subsequent tries? It would be unexpected for tucking to never return so there should be some retry limit, and an error or success should be returned as in the other methods.